tests/test042.cpp

```diff
@@ -0,0 +1,50 @@
+// test042.cpp - read column values as char
```

Comment:
This test only tests reading. It would be good to also add a test performing writing.

Reply:
Each new test case need to be added to CMakeLists.txt in order to build / run automatically.